<launch>

    <node pkg="aruco_detect" type="marker_detect" name="marker_detect" output="screen">
        <param name="camera_topic" type="string" value="/usb_cam/image_raw" />
        <param name="camera_info" type="string" value="/home/yjd-nx/shiyan/src/aruco_detect/config/camera_param.yaml" />
    </node>

</launch>